client/v2/autocli/flag/address.go

```diff
@@ -2,22 +2,43 @@ package flag
 
 import (
 	"context"
-
+	reflectionv2alpha1 "cosmossdk.io/api/cosmos/base/reflection/v2alpha1"
+	"github.com/cosmos/cosmos-sdk/types"
 	"google.golang.org/protobuf/reflect/protoreflect"
 )
 
 type addressStringType struct{}
 
-func (a addressStringType) NewValue(_ context.Context, _ *Builder) Value {
-	return &addressValue{}
+func (a addressStringType) NewValue(ctx context.Context, b *Builder) Value {
+
+	if b.AddressPrefix == "" {
+		conn, err := b.GetClientConn()
+		if err != nil {
+			panic(err)
+		}
+		reflectionClient := reflectionv2alpha1.NewReflectionServiceClient(conn)
+		if ctx == nil {
```

Comment:
I don't think we should overwrite a nil context. It may be hard to debug for the caller.

Reply:
What do you sugest I use instead to avoid possible error because I was getting a nil pointer from the ctx being nil?

Reply:
I suggest just passing it. The caller need to ensure ctx isn't nil.

Reply:
Oh ok perfect, I got a panic from it while testing I will dig into the caller to check what is up